app/services/eventful_service.rb

```diff
@@ -0,0 +1,10 @@
+class EventfulService
+  def initialize
+  end
+
+  def events(location)
+    response = Faraday.get("http://api.eventful.com/json/events/search?&location=#{location}&sort_order=popularity&app_key=#{ENV["EVENTFUL_API_KEY"]}&date=march&sort_direction=descending&page_size=16&include=popularity")
+    raw_events = JSON.parse(response.body)["events"]["event"]
+    raw_events.map { |event| Event.new(event) }
```

Comment:
Instead of having the `EventfulService` create instances of `Event` have the `Event` class call out to `EventfulService` and instantiate `Event`s in the `Event` class. The big reason is it's common to share services between multiple apps even within the same organization. When you find that happening it's usually time to break out the service into a gem. When that happens we don't want our service to be dependent on unnecessary classes (in this case `Event`).